drf_spectacular/settings.py

```diff
@@ -84,6 +84,10 @@
     'OAUTH2_TOKEN_URL': None,
     'OAUTH2_REFRESH_URL': None,
     'OAUTH2_SCOPES': None,
+
+    # Dictionary of configurations to pass to the SwaggerUI({ ... })
+    # https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/
+    'SWAGGER_SETTINGS': {}
```

Comment:
can you call it `SWAGGER_UI_SETTINGS` and move it below the `SERVE_*` settings as they are closely related?